atlite/cutout.py

```diff
@@ -302,26 +310,31 @@ def transform_r(self):
 
     @property
     def dx(self):
+        """Spatial resolution on the x axis."""
```

Comment:
```suggestion
        """Spatial resolution on the x coordinate (longitude)."""
```

`x` is always implicitly longitude, right?

Reply:
not necessarily, atm we are only using the latlon projection, but other projections would be possible...

Reply:
Gotcha. Is 'coordinate' or 'axis' more comprehensive?

Reply:
indeed!